custom-elements/connected-callbacks-html-fragment-parsing.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Custom Elements: connectedCallback not called during HTML fragment parsing</title>
+<meta name="author" title="Rob Buis" href="mailto:rbuis@igalia.com">
+<meta name="assert" content="connectedCallback must not be called during HTML fragment parsing">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/parsing.html#create-an-element-for-the-token">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/custom-elements-helpers.js"></script>
+</head>
+<body>
+<div id="log"></div>
+<script>
+
+var iteration = 0;
+
+document_types().forEach(function (entry) {
+    let documentName = entry.name;
+    let getDocument = entry.create;
```

Comment:
These two variables are only used once. There is no need to define them as local variables.